server/src/main/java/org/opensearch/action/admin/cluster/remotestore/stats/RemoteStoreStatsRequestBuilder.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.remotestore.stats;
+
+import org.opensearch.action.support.broadcast.BroadcastOperationRequestBuilder;
+import org.opensearch.client.OpenSearchClient;
+import org.opensearch.common.unit.TimeValue;
+
+/**
+ * Builder for RemoteStoreStatsRequest
+ *
+ * @opensearch.internal
+ */
+public class RemoteStoreStatsRequestBuilder extends BroadcastOperationRequestBuilder<
+    RemoteStoreStatsRequest,
+    RemoteStoreStatsResponse,
+    RemoteStoreStatsRequestBuilder> {
+
+    public RemoteStoreStatsRequestBuilder(OpenSearchClient client, RemoteStoreStatsAction action) {
+        super(client, action, new RemoteStoreStatsRequest());
+    }
+
+    /**
+     * Sets timeout of request.
+     */
+    public final RemoteStoreStatsRequestBuilder setTimeout(TimeValue timeout) {
```

Comment:
setTimeout and setLocal does not seem to be getting used. Any reasons for having these right now?

Reply:
just kept the basic setters, not used currently.